src/interactive.js

```diff
@@ -0,0 +1,9 @@
+export default function interactive(e) {
+    if (e.target.checked) {
+      e.target.nextElementSibling.classList.add('complete');
+      console.log("complete");
```

Comment:
- [ ] [OPTIONAL] It is not best practice to ship your code with `console.log` code. [Read here about ](https://bigfont.ca/do-not-put-console-log-into-version-control-or-production/)why this is important.